content/tutorials/sse-and-http-streaming.textile

```diff
@@ -21,7 +21,7 @@ tags:
     - SSE
 ---
 
-Server-Sent Events (SSE) is a standard describing how servers can initiate data transmission towards clients once an initial client connection has been established. They are commonly used to send message updates or continuous data streams to a browser client and are designed to enhance native, cross-browser streaming through a JavaScript API called "EventSource":https://en.wikipedia.org/wiki/Server-sent_events. Using this API, a client can easily receive an event stream by sending a request to a particular URL endpoint. Basically, using SSE, a client can receive continuous event updates without needing to use any client library SDK or even an adapter like "MQTT":https://ably.com/documentation/mqtt.
+Server-Sent Events (SSE) is a standard describing how servers can initiate data transmission towards clients once an initial client connection has been established. They are commonly used to send message updates or continuous data streams to a browser client and are designed to enhance built-in, cross-browser streaming through a JavaScript API called "EventSource":https://en.wikipedia.org/wiki/Server-sent_events. Using this API, a client can easily receive an event stream by sending a request to a particular URL endpoint. Basically, using SSE, a client can receive continuous event updates without needing to use any client library SDK or even an adapter like "MQTT":https://ably.com/documentation/mqtt.
```

Comment:
I would suggest that you could remove "built-in" here, because you are already talking about the capabilities of browser clients. @tbedford - what do you think?

Reply:
Yes, we can remove built-in. Also let's remove "Basically,".

> any client library SDK

an Ably Client Library SDK

> or even an adapter like

or an adapter such as

Reply:
Removed "built-in". Other changes can be addressed in content review for that page.